mu-plugins/loader.php

```diff
@@ -5,6 +5,7 @@
 
 require_once __DIR__ . '/blocks/global-header-footer/blocks.php';
 require_once __DIR__ . '/global-fonts/index.php';
+require_once __DIR__ . '/plugin-tweaks/index.php';
```

Comment:
> It started out as https://github.com/WordPress/pattern-directory/pull/463 for the pattern directory, but after some discussion it seems better to put it in mu-plugins since it will benefit any use of the Internal Notes plugin anywhere on wporg.

I'm leery of trying too hard to support 3rd party usage after our experience w/ CampTix. If we put this directly in the Notes plugin, then 3rd parties could disable it w/ `remove_filter()`.

IMO it'd be better to not have tweaks for 1st party plugins, b/c I worry I'll forget they're there and only look in the main plugin itself.

I don't feel strongly, though, whatever you decide WFM.